build.gradle

```diff
@@ -41,8 +41,13 @@ dependencies {
 }
 
 javafx {
-    version = "11.0.1"
-    modules = [ 'javafx.controls' ]
+    version = "11.0.2"
+    modules = [ 'javafx.controls',
```

Comment:
we don't need all 6 of these as far as I am aware: across all of our samples we only use:
'javafx.controls',
'javafx.fxml',
'javafx.web',
'javafx.media'

I believe .base and .graphics are used internally though but not needed for building an application.

Reply:
I believe base and graphics are included by using something like controls so don't necessarily need to be explicitly added.